rllib/evaluation/worker_set.py

```diff
@@ -139,6 +140,17 @@ def __init__(
                     or not trainer_config.get("action_space")
                 )
             ):
+                # First assert that the first worker is ready.
+                # Not doing so leads to worker-not-ready errors sometimes on Windows.
+                for _ in range(10):
+                    try:
+                        test = ray.get(self.remote_workers()[0].apply.remote(lambda w: w.worker_index))
+                        assert test == 1
+                        break
+                    except Exception as e:
```

Comment:
this try catch is unnecessary -- ray.get will block until the return values are ready for all of the remote calls that you have passed to ray.get

Reply:
Normally, yes, but Windows is the problem, it seems.